package.json

```diff
@@ -50,7 +56,7 @@
     "ember-source": "~2.13.0",
     "eslint": "^4.11.0",
     "eslint-config-prettier": "^2.7.0",
-    "eslint-plugin-prettier": "^2.3.1",
+    "eslint-plugin-prettier": "2.6.0",
```

Comment:
Again this is there because of this issue https://github.com/ember-cli/ember-cli-eslint/issues/235